js/constants/appConfig.js

```diff
@@ -180,6 +180,8 @@ module.exports = {
     'sync.type.history': false,
     'sync.type.siteSetting': true,
     'general.downloads.default-save-path': null,
+    // BATads
+    'BATads.enabled': true, // need to default it to off for merge into master
```

Comment:
switch to false, before merge!!

Reply:
OK, will do. 
I'm assuming the purpose of this PR is to ultimately merge back into main branch?
I apologize for my ignorance; most of my career I'm out working on my own repo, which eventually gets used by clients.

Reply:
Yes purpose of this PR is to track progress, add comments for others and prepare code for the review. When review is done and code (PR) is approved we will merge it to the appropriate version.